creator-node/src/routes/tracks.js

```diff
@@ -37,12 +40,90 @@ const RehydrateIpfsQueue = require('../RehydrateIpfsQueue')
 const { FileProcessingQueue } = require('../FileProcessingQueue')
 const DBManager = require('../dbManager')
 const { generateListenTimestampAndSignature } = require('../apiSigning.js')
+const DiskManager = require('../diskManager')
 
+const server = new tus.Server()
 const readFile = promisify(fs.readFile)
+const tmpTrackArtifactsPath = DiskManager.getTmpTrackUploadArtifactsPath()
+const resumableUploadRoute = path.join(tmpTrackArtifactsPath, '*', '*')
+
+// Structure: <uuid>.<file_extension>. e.g. 1234-1234-1234-1234.mp3
+const getFileName = (req) => {
```

Comment:
lets pass in the params directly instead of req?

Reply:
yup

Reply:
ah actually, we can't do that because this fn is passed into the FileStore ctor and requires the req object

Reply:
oh i see. okay this isn't a huge thing so resolving